package.json

```diff
@@ -1,25 +1,34 @@
 {
   "private": true,
+  "workspaces": [
+    "packages/fetch",
+    "packages/fetch-cached-dns",
+    "packages/fetch-retry"
+  ],
   "scripts": {
     "lint": "turbo run lint",
     "test": "turbo run test",
     "prettier": "prettier -w .",
     "prepare": "husky install",
-    "eslint": "cross-env TIMING=1 eslint --max-warnings 0 --config .eslintrc.js"
+    "eslint": "cross-env TIMING=1 eslint --max-warnings 0 --config .eslintrc.js",
+    "changeset": "changeset",
+    "release": "changeset publish"
```

Comment:
Use:  "release": "pnpm build && changeset publish",

Reply:
No build command in this repo. Still just javascript files